module/validation/seal_validator_test.go

```diff
@@ -121,7 +112,12 @@ func (s *SealValidationSuite) TestSealInvalidAggregatedSigCount() {
 //   ... <- LatestSealedBlock <- B0 <- B1{ Result[B0], Receipt[B0] } <- B2 <- ░newBlock{ Seal[B0]}░
 // The gap of 1 block, i.e. B2, is required to avoid a sealing edge-case
 // (see test `TestSeal_EnforceGap` for more details)
-func (s *SealValidationSuite) TestSealEmergencySeal() {
+func (s *SealValidationSuite) TestSealEmergencySeal_VerificationRequire0ApprovalEmergencyNotTriggered() {
```

Comment:
I think the godoc is outdated for this function, these parts:
> and that the emergency-seal metric will be incremented.

>  seal which has 0 signatures for at least one chunk will be accepted

We're checking that emergency sealing is not triggered on line 128 and the seal we're testing has 2 approvals (line 124)